src/scss/components/dropdown-select/_mixins.scss

```diff
@@ -1,12 +1,39 @@
-@import "mixins/microsoft";
-@import "mixins/mozilla";
-@import "mixins/standard";
-@import "mixins/webkit";
-
-@mixin reset-select { // less: #Reset.select()
-  @include ms-reset-select;
-  @include moz-reset-select;
-  @include webkit-reset-select;
-  @include std-reset-select;
+@mixin __reset {
```

Comment:
Having "private" mixins with the same name (across different files) _shouldn't_ be an issue because we use them in the same file as they're defined.  If for some reason this behavior ever changes, we can rename the private mixins without affecting how the "public" mixins are consumed.